### ISSUE TITLE
Multiagent visuallizer is broken

### ISSUE THREAD

**Issue by eugenevinitsky** [flow-project/flow#487]
Ray has updated their visuallizer to work correctly with multi-agent and LSTMs. We need to port it over:
https://github.com/ray-project/ray/blob/master/python/ray/rllib/rollout.py

**Comment by AboudyKreidieh** [comment]
@eugenevinitsky is this still an issue? If so, we should assign it to someone

**Comment by eugenevinitsky** [comment]
Yeah, there's a tiny issue here. It's mostly fixed, but, we don't pass the previous reward to the LSTM which can be an issue if the LSTM was trained with previous reward as an input.
